### PR TITLE
Feat/tb

### PR DESCRIPTION
**Pull request by yilinxu**

### CODE REVIEW

File: gdcdictionary/schemas/imaging_file.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  - created_datetime
+  - updated_datetime
+
+links:
+  - name: imaging_study
+    backref: imaging_files
+    label: data_for
+    target_type: imaging_file
+    multiplicity: many_to_one
+    required: TRUE
```
Comment (philloooo) [inline gdcdictionary/schemas/imaging_file.yaml]: Let’s use consistent capitalization for Boolean? All lowercase?

File: gdcdictionary/schemas/treatment.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@ systemProperties:
 links:
   - name: diagnoses
```
Comment (philloooo) [inline gdcdictionary/schemas/treatment.yaml]: Are there existing data using this? If so we will need to migrate the data?

Comment (yilinxu) [reply gdcdictionary/schemas/treatment.yaml]: yes, existing data are there

File: gdcdictionary/schemas/imaging_file.yaml (hunk 1 above)
Comment (yilinxu) [reply gdcdictionary/schemas/imaging_file.yaml]: sure, will do

**Pull request merged**